hack/build.sh

```diff
@@ -284,6 +284,8 @@ cross_build() {
   GOOS=darwin GOARCH=amd64 go build -mod=vendor -ldflags "${ld_flags}" -o ./kn-darwin-amd64 ./cmd/... || failed=1
   echo "   🎠 kn-windows-amd64.exe"
   GOOS=windows GOARCH=amd64 go build -mod=vendor -ldflags "${ld_flags}" -o ./kn-windows-amd64.exe ./cmd/... || failed=1
+  echo "   Z  kn-linux-s390x"
```

Comment:
```suggestion
  echo "   🗡️  kn-linux-s390x"
```

We love emojis ;-) What's about a dagger like the one that [Zorro](https://en.wikipedia.org/wiki/Zorro) used ?

Reply:
Thanks for the suggestion. I love a dagger, but let me check that with my manager first if the emoji, which does not representing the Z system, can be used in the open source. (actually I spent more time on choosing a suitable emoji than the PR itself and ended up using the alphabet, which is not a proprietary asset)

Could you please hold this PR until I get an answer for this?

Reply:
sure. btw, we are using also non official emoji for all the other OS, and the emoji is not meant to represent an official product logo, but just as kind of a fancy "bullet point" in the list. We also never talk about the architecture itself here, as you see it's just the name of the binary that we produce.

Reply:
Yop, I have the same opinion. I am just wondering if the company thinks the same. ^^

Reply:
I got the decision from the company about the emoji: just sticking to the capital z as it is. Is it okay to go with it?